tools/boards.txt.py

```diff
@@ -1119,12 +1119,11 @@ def flash_map (flashsize_kb, spiffs_kb = 0):
         ( menu + '.upload.maximum_size', "%i" % max_upload_size ),
         ( menub + 'rfcal_addr', "0x%X" % rfcal_addr)
         ])
-    #if spiffs_kb > 0:
```

Comment:
Per @earlephilhower : putting this if back improves the error message for 4M0

Reply:
fixed